src/main/java/com/github/thedeathlycow/moregeodes/features/ModConfiguredFeatures.java

```diff
@@ -35,6 +34,7 @@ public class ModConfiguredFeatures {
 
     public static final RegistryEntry<ConfiguredFeature<?, ?>> GYPSUM_CRYSTALS;
     public static final RegistryEntry<ConfiguredFeature<?, ?>> GYPSUM_PATCH;
+    public static final RegistryEntry<ConfiguredFeature<?, ?>> CERTUS_GEODE;
```

Comment:
This should be marked as `@Nullable` explicitly

Reply:
True